src/org/chartjs/Bar.js

```diff
@@ -14,13 +14,15 @@ foam.CLASS({
     },
     function updateChart_(data) {
       // TODO: Support multiple datasets via nested groupby
+      var colors = this.colors;
       var groups = data.groups;
       var keys = data.sortedKeys();
       var data = {
         labels: keys,
         datasets: [
           {
             label: data.arg2.label || data.arg2.model_.label,
+            backgroundColor: colors,
```

Comment:
I don't think we want to pass the colours here, we don't want each bar to have a different colour, we want each dataset to have a unique colour for when we plot a GROUP_BY(GROUP_BY()).

Reply:
good call. Just using a single color for now and when we support nested group bys I'll add more colors.